boards/BMS/Src/App/App_Accumulator.c

```diff
@@ -165,7 +243,7 @@ void App_Accumulator_RunOnTick100Hz(struct Accumulator *const accumulator)
     {
         case GET_CELL_VOLTAGE_STATE:
 
-            UPDATE_PEC15_ERROR_COUNT(accumulator->read_cell_voltages(), accumulator->num_comm_tries)
+            App_Accumulator_ReadCellVoltage(accumulator);
 
             // Write to configuration register to configure cell discharging
             // TODO: re-enable
```

Comment:
The state variable here will always be GET_CELL_VOLTAGE_STATE - is that intentional?

Reply:
(out of scope of this PR)